src/status_im2/subs/wallet.cljs

```diff
@@ -0,0 +1,15 @@
+(ns status-im2.subs.wallet
```

Comment:
I think wallet is quite a lot of subs and perhaps we need to split them up a bit. Perhaps we use 
`status-im2.subs.wallet.wallet` and then we have other parts of the wallet subs in split files.
e.g 
`status-im2.subs.wallet.assets`
`status-im2.subs.wallet.accounts`

I'm okay with one file too and we can later split it up, however I think we should do that here if that is to be the case as there is already some wallet subs defined in `status-im2.subs.wallet.*`

Reply:
@J-Son89, thanks! I didn't notice we already have a wallet folder for subs)